sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -258,6 +258,7 @@
 
         // Client telemetry header
         public static final String DATABASE_ACCOUNT_NAME = "x-ms-databaseaccount-name";
+        public static final String ENVIRONMENT_NAME = "x-ms-environment-name";
```

Comment:
To avoid clashing these headers with backend, we should rename both of them with some java sdk prefix or suffix. 
like, `x-ms-java-sdk-environment-name` and `x-ms-java-sdk-databaseaccount-name`
or 
`x-ms-client-telemetry-environment-name` and `x-ms-client-telemetry-databaseaccount-name`
or something like that.

Reply:
Discussed offline this if for Juno backend service so header name have to follow the existing convention